packages/backend/src/boot/master.ts

```diff
@@ -110,8 +107,9 @@ function showNodejsVersion(): void {
 
 	nodejsLogger.info(`Version ${runningNodejsVersion.join('.')}`);
 
-	if (!satisfyNodejsVersion) {
-		nodejsLogger.error(`Node.js version is less than ${requiredNodejsVersion.join('.')}. Please upgrade it.`, null, true);
+	const minVersion = 'v16.14.0';
+	if (semver.lt(process.version, minVersion)) {
+		nodejsLogger.error(`Please make sure you've installed and are using NodeJS >=${minVersion}! You're currently on NodeJS ${process.version}.`);
```

Comment:
No need to print the running version again. I'd also rephrase it to be more straight to the point since this is a logfile.
```suggestion
		nodejsLogger.error(`At least Node.js ${minVersion} required!`);
```